luigi/scheduler.py

```diff
@@ -234,6 +234,27 @@ def load(self):
         else:
             logger.info("No prior state file exists at %s. Starting with clean slate", self._state_path)
 
+    def cascade_disabled_state(self, tasks):
+
+        """ Iterate through all pending tasks, setting the status to DISABLED
+        if at least one of it's dependencies is DISABLED and the rest are
+        either DONE or DISABLED. """
+
+        for task in tasks:
```

Comment:
We don't need to worry about that because this method gets evaluated in every loop of the scheduler. So we may not disable all tasks on the first pass, but we will eventually (within a couple of passes).